File: solved.ac/CLASS1_JS/1000.js
const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];
rl.on('line', (line) => {
  input = line.split(' ').map((elem) => parseInt(elem));
  console.log(input[0] + input[1]);
  rl.close();
})

File: boj-rust/1000.rs
use std::io;

fn main(){
  let mut input = String::new();

  io::stdin().read_line(&mut input)
        .expect("Falied to read line");

  let numbers: Vec<_> = input
    .split(" ")
    .map(|num| num.trim().parse::<i32>().unwrap())
    .collect();

  println!("{}", numbers[0] + numbers[1]);
}